src/components/deploymentConfig/constants.tsx

```diff
@@ -85,3 +87,11 @@ export const EDITOR_VIEW = {
     BASIC: 'BASIC',
     ADVANCED: 'ADVANCED',
 }
+
+export const chartTypeTabKeys = { DEVTRON_CHART: 'devtronChart', CUSTOM_CHARTS: 'customCharts' }
+export const chartTypeTab = { devtronChart: 'Charts by Devtron', customCharts: 'Custom charts' }
+export const chartDocumentationLink = {
+    'CronJob & Job': DOCUMENTATION.JOB_CRONJOB,
+    'Rollout Deployment': DOCUMENTATION.ROLLOUT,
+}
+export const recommendedChartName = 'Deployment'
```

Comment:
Create const in all Upper cases with `_` in between for multi-word names.

Reply:
Done